phys/module_physics_init.F

```diff
@@ -1078,6 +1081,9 @@ SUBROUTINE phy_init ( id, config_flags, DT, restart, zfull, zhalf,     &
       SNOWNCV(i,j)=0.
       GRAUPELNCV(i,j)=0.
       ACSNOW(i,j)=0.
+      LPOS(i,j)=0.
```

Comment:
@BarryHLynn 
Barry,
Since you set the values of LPOS, LNEG, LNEU to zero, they should not be INTENT(INOUT), right? INTENT(OUT) only?